app/ui-react/syndesis/src/modules/data/pages/VirtualizationViewsPage.tsx

```diff
@@ -99,206 +100,247 @@ export class VirtualizationViewsPage extends React.Component<
+                          >
+                            {t('shared:DataVirtualizations')}
+                          </Link>
+                          <span>
+                            {virtualizationId + ' '}
+                            {t('data:virtualization.views')}
+                          </span>
+                        </Breadcrumb>
+                        <ViewHeader
+                          i18nTitle={virtualization.keng__id}
+                          i18nDescription={virtualization.tko__description}
+                        />
+                        <WithViewEditorStates
+                          idPattern={virtualization.serviceVdbName + '*'}
+                        >
+                          {({ data, hasData, error, read }) => {
                             return (
-                              <WithListViewToolbarHelpers
-                                defaultFilterType={filterByName}
-                                defaultSortType={sortByName}
-                              >
-                                {helpers => {
-                                  const viewDefns = data.map(
-                                    (editorState: ViewEditorState) =>
-                                      editorState.viewDefinition
-                                  );
-                                  const filteredAndSorted = getFilteredAndSortedViewDefns(
-                                    viewDefns,
-                                    helpers.activeFilters,
-                                    helpers.currentSortType,
-                                    helpers.isSortAscending
-                                  );
+                              <WithVirtualizationHelpers>
+                                {({ deleteView }) => {
+                                  const handleDeleteView = async (
+                                    viewName: string
+                                  ) => {
+                                    try {
+                                      await deleteView(
+                                        virtualization,
+                                        viewName
+                                      ).then(read);
+                                      pushNotification(
+                                        t('virtualization.deleteViewSuccess', {
+                                          name: viewName,
+                                        }),
+                                        'success'
+                                      );
+                                    } catch (error) {
+                                      const details = error.message
+                                        ? error.message
+                                        : '';
+                                      pushNotification(
+                                        t('virtualization.deleteViewFailed', {
+                                          details,
```

Comment:
I don't think the details here are specified correctly...  should probably be something like details: detailContent ?

Reply:
details looks correct since it's been declared to equate to error.message